src/test/custom/identity/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if(${TRITON_ENABLE_GPU})
 target_include_directories(identity PRIVATE ${CUDA_INCLUDE_DIRS})
 target_link_libraries(
   identity
-  PUBLIC ${CUDA_nvml_LIBRARY}
```

Comment:
Does this really need to link against DCGM (or previously NVML)?

Reply:
No, reverted to NVML

Reply:
You shouldn't need to link either. We no longer use nvml at all, and I don't know why the legacy custom backend was linking against it in the first place...